.github/workflows/build_test.yml

```diff
@@ -19,6 +19,21 @@ env:
   PYTHON_VERSION: 3.7
 
 jobs:
+  lint:
+    if: "github.repository == 'numpy/numpy' && !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip github]')"
+    runs-on: ubuntu-latest
+    env:
+      DO_LINT: 1
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - uses: actions/setup-python@v2
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}
+    - uses: ./.github/actions
+
```

Comment:
Rather than use the github/actions and modify the misnamed travis*.sh scripts, can you just do the check here?